src/ducks/reimbursements/NoPendingReimbursements.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import Padded from 'components/Padded'
+import { getCategoryName } from 'ducks/categories/categoriesMap'
+import Caption from 'ducks/reimbursements/Caption'
+
+const NoPendingReimbursements = ({ period, doc }) => {
+  const { t } = useI18n()
+
+  const categoryName = doc.categoryId ? getCategoryName(doc.categoryId) : null
+  const message = categoryName
+    ? t(`Reimbursements.noPending.${categoryName}`, { period })
+    : t('Reimbursements.noPending.generic', { period })
+
+  return (
+    <Padded className="u-pv-0">
+      <Caption>{message}</Caption>
+    </Padded>
+  )
```

Comment:
let's add a test on this new component please to document its usage.

Reply:
je ne pense pas que ce soit la peine non plus ici vu ce que fait le composant